src/Handler/Project/Drop/DropProjectHandler.php

```diff
@@ -35,18 +35,41 @@ public function __invoke(
 
         $iamService = $this->clientManager->getIamClient($credentials);
         $serviceAccountsService = $iamService->projects_serviceAccounts;
+        $commandMeta = $command->getMeta();
+        if ($commandMeta !== null) {
+            // override root user and use other database as root
+            $commandMeta = $commandMeta->unpack();
+            assert($commandMeta instanceof DropProjectCommand\DropProjectBigqueryMeta);
+            $fileStorageBucketName = $commandMeta->getGcsFileBucketName();
+        } else {
+            throw new Exception('CreateProjectBigqueryMeta is required.');
+        }
```

Comment:
kill the else
```suggestion
        if ($commandMeta === null) {
          throw new Exception('CreateProjectBigqueryMeta is required.');
        }
          // override root user and use other database as root
          $commandMeta = $commandMeta->unpack();
          assert($commandMeta instanceof DropProjectCommand\DropProjectBigqueryMeta);
          $fileStorageBucketName = $commandMeta->getGcsFileBucketName();
```

Reply:
Navíc nechápu ten komentář `override root user and use other database as root`

Reply:
💀